README.md

```diff
@@ -213,39 +213,39 @@ and improve control.
 But integrating and maintaining multiple processors needs a lot of dev effort.
 Why should devs across companies repeat the same work?
 Why can't it be unified and reused? Hence, HyperSwitch was born to create that
-reusable core and let companies build and customize on top of it.
+reusable core and let companies build and customise it on top of it.
```

Comment:
The `it` here seems unnecessary

Reply:
> The `it` here seems unnecessary

made it more clear as per the subject of the sentence.